sqlite-android/build.gradle

```diff
@@ -18,14 +18,11 @@ gradle.projectsEvaluated {
 }
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 26
+    buildToolsVersion "26.0.1"
 
     defaultConfig {
-        minSdkVersion 9
-        targetSdkVersion 25
-        versionCode 1
-        versionName project.version
```

Comment:
is there a reason this is removed? the version name should be available as part of the library build info I believe

Reply:
The versionName removal was accidental, added it back.